test/solc/CommandLineInterface.cpp

```diff
@@ -1018,6 +1025,72 @@ BOOST_AUTO_TEST_CASE(cli_include_paths)
 	BOOST_TEST(result.reader.basePath() == expectedWorkDir / "base/");
 }
 
+BOOST_AUTO_TEST_CASE(enum_only)
+{
+	TemporaryDirectory tempDir({"base/"}, TEST_CASE_NAME);
+	TemporaryWorkingDirectory tempWorkDir(tempDir);
+
+	string const mainContractSource =
+		"// SPDX-License-Identifier: GPL-3.0\npragma solidity >=0.0;\n\nenum Status {\n		test\n}\n";
+
+	createFilesWithParentDirs({tempDir.path() / "base/main.sol"}, mainContractSource);
+
+	boost::filesystem::path canonicalWorkDir = boost::filesystem::canonical(tempDir);
+	boost::filesystem::path expectedWorkDir = "/" / canonicalWorkDir.relative_path();
+
+	vector<string> commandLine = {
+		"solc",
+		"--no-color",
+		"--base-path=base/",
+		"base/main.sol",
+	};
+
+	CommandLineOptions expectedOptions;
+
+	string const expectedStdoutContent = "Compiler run successful. No contracts to compile.\n";
+	OptionsReaderAndMessages result = runCLI(commandLine, "");
+
+	BOOST_TEST(result.stderrContent == "");
+	if (SemVerVersion{string(VersionString)}.isPrerelease())
+		BOOST_TEST(result.stdoutContent == "");
+	else
+		BOOST_TEST(result.stdoutContent == expectedStdoutContent);
+	BOOST_REQUIRE(result.success);
```

Comment:
There's quite a bit of unnecessary stuff here. For example we don't really need base path in this test. In fact, we don't even need a temporary dir and we can just pass the source to standard input.

Also, you have some unused variables (`expectedWorkDir`, `expectedOptions`). I wonder why the compiler is not complaining about them.

```suggestion
	string const contractSource = R"(
		// SPDX-License-Identifier: GPL-3.0
		pragma solidity >=0.0;

		enum Status { test }
	)";

	string const expectedStdoutContent = "Compiler run successful. No contracts to compile.\n";
	OptionsReaderAndMessages result = runCLI({"solc", "-"}, contractSource);

	BOOST_TEST(result.stderrContent == "");
	if (SemVerVersion{string(VersionString)}.isPrerelease())
		BOOST_TEST(result.stdoutContent == "");
	else
		BOOST_TEST(result.stdoutContent == expectedStdoutContent);
	BOOST_REQUIRE(result.success);
```

Please simplify the other test in a similar way.

Reply:
Simplified these tests.